### ISSUE TITLE
Glowing colors not working on NPCs created with code.

### ISSUE THREAD

**Issue by null321-jared** [CitizensDev/Citizens2#2166]
The output of command `/version` on my server is:
```
[23:14:54 INFO]: This server is running Paper version git-Paper-318 (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
[23:14:54 INFO]: Checking version, please wait...
[23:14:55 INFO]: You are running the latest version
```
Though it's the same on Spigot.

The output of command `/version citizens` on my server is:
`Citizens version 2.0.26-SNAPSHOT (build 1970)`

The server and client are 1.15.2. There are no version support plugins installed on this server.
It's a server running in offline mode on a bungeecord in online mode.
Logs (which are not really relevant here): https://pastebin.com/9tvrjkM9

I am creating an NPC with
`CitizensAPI.getNPCRegistry().createNPC(EntityType.PLAYER, name);`

The name is simply "Raider".
I cannot, using code or the commands, change the color of the Raider NPC, with one exception. That exception is creating another NPC with the same name and applying the color to it with the commands.

I don't think there is anything installed that would conflict with the packets or scoreboard teams.

Here is an image of the issue:
![image](https://user-images.githubusercontent.com/12385687/82974603-5e155580-9fa8-11ea-947c-c2675f949af3.png)

**Comment by mcmonkey4eva** [comment]
From your logs:
- remove PlugMan (it breaks things)
- Fix luckperms (it's failing to load in your logs)
- Get rid of `RemoteToolkitPlugin` (it's not loading, and google says that's a plugin from more than 5 years ago that was never updated)

----

Enable glowing, *then* set the color.

Works fine testing on my local server: https://i.alexgoodwin.media/i/misc/1ef6c3.gif

**Comment by jaredoconnell** [comment]
> 
> 
> From your logs:
> 
>     * remove PlugMan (it breaks things)
> 
>     * Fix luckperms (it's failing to load in your logs)
> 
>     * Get rid of `RemoteToolkitPlugin` (it's not loading, and google says that's a plugin from more than 5 years ago that was never updated)
> 
> 
> Enable glowing, _then_ set the color.
> 
> Works fine testing on my local server: https://i.alexgoodwin.media/i/misc/1ef6c3.gif

Does PlugMan break things if you don't use the commands? I use it for my in-development plugins, which don't break.
I updated LuckPerms (though I don't see how it's relevant to this)
RemoteToolkitPlugin is working (enough) and is needed for the wrapper that runs the server.

I did it in the order you stated, and it does not work on this NPC. It does work when I do it like you did, using the commands.
![image](https://user-images.githubusercontent.com/46976761/83075070-41792c00-a041-11ea-8c5d-746be201c3f7.png)
Other details include there being several NPCs of the same name.

Here is a video of it happening: https://youtu.be/upBVcbpQstA
